src/sentry/integrations/github/client.py

```diff
@@ -659,6 +666,32 @@ def get_blame_for_file(
 
         return ref.get("target", {}).get("blame", {}).get("ranges", [])
 
+    def get_blame_for_files(self, files: Sequence[SourceLineInfo]) -> Sequence[FileBlameInfo]:
+        file_path_mapping = generate_file_path_mapping(files)
+
+        try:
+            response = self.post(
+                path="/graphql",
+                data={"query": create_blame_query(file_path_mapping)},
+                allow_text=False,
+            )
+        except ValueError as e:
+            sentry_sdk.capture_exception(e)
+            return []
+
+        if not isinstance(response, MappingApiResponse):
+            raise ApiError("Response is not JSON")
+
+        if response.get("errors"):
+            err_message = ", ".join(
+                [error.get("message", "") for error in response.get("errors", [])]
+            )
+            logger.info("GitHub blame response had some errors", extra={"error": err_message})
```

Comment:
If you do `logger.error` you would see it on Sentry and can determine if there's something you did in the call that needs your attention.